test/model/friendmessagedispatcher_test.cpp

```diff
@@ -170,16 +114,13 @@ void TestFriendMessageDispatcher::init()
 {
     f = std::unique_ptr<Friend>(new Friend(0, ToxPk()));
     f->setStatus(Status::Status::Online);
-    f->onNegotiationComplete();
     messageSender = std::unique_ptr<MockFriendMessageSender>(new MockFriendMessageSender());
-    coreExtPacketAllocator =
-        std::unique_ptr<MockCoreExtPacketAllocator>(new MockCoreExtPacketAllocator());
     sharedProcessorParams = std::unique_ptr<MessageProcessor::SharedParams>(
```

Comment:
we are at least c++14, so all `std::unique_ptr<T>(new T)` should be replace with `std::make_unique<T>()`

but that is for another pr.

Reply:
https://github.com/TokTok/qTox/pull/76 yes, this one. Review at your leisure.

Reply:
ah yis